scripts/publish_binary_size.js

```diff
@@ -130,4 +130,4 @@ github.apps.createInstallationToken({installation_id: SIZE_CHECK_APP_INSTALLATIO
     .then(({data}) => {
         github.authenticate({type: 'token', token: data.token});
         return query();
-    });
```

Comment:
Here's an approach to use a single publish script:
1. Introduce an object `sizeCheckInfo` to record the results from size checks on the commits
1. On line 116, instead of creating and returning a new `putObject` promise, populate the `sizeCheckInfo` object with all the appropriate information from the `row`.
1. On line 132, chain a `then` clause onto `query`. Here, call two methods (1) to publish the binary-size metrics to the `mapbox` bucket, and (2) to publish the metrics to the `mapbox-load-dock` bucket in the respective formats.